infrastructure/lambda/template.yaml

```diff
@@ -900,6 +923,238 @@ Resources:
             Period: 300
             Stat: Sum
 
+  KBVQuestionFunctionCanary5xxErrors:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
```

Comment:
if this is set to true it will trigger alarms during deployment and we'll get notifications in slack